asf_heat_pump_suitability/pipeline/run_scripts/run_make_webmap_geojsons.py

```diff
@@ -0,0 +1,240 @@
+"""
+Create geojson files to use as inputs to tippecanoe map tiler for the heatpump suitability webmap.
+
+This script gets the relevant England & Wales LSOA and Scotland DataZone boundaries from the /mapdata/ prefix in
+the heat pump suitability s3 bucket. Merges the geometries. Joins the suitability scoring (provided via config or as a flag)
```

Comment:
```suggestion
the heat pump suitability s3 bucket. Concatenates the geometries. Joins the suitability scoring (provided via config or as a flag)
```
Is this meant to be concats?

Reply:
Sure.